shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/antlr4/imports/sharding/RALStatement.g4

```diff
@@ -44,5 +44,5 @@ tableName
     ;
 
 shardingValue
-    : IDENTIFIER
+    : INT
```

Comment:
Could a shardingValue equal `aaa`?

Reply:
No. I confirmed with @RaigorJiang  that this can only be a 'INT'.

Reply:
After discussion, we think it may not only be int.